client/package.json

```diff
@@ -30,7 +30,10 @@
         "electron:release": "npm run electron:copy && cd electron && npm run electron:release",
         "electron:debug": "npm run electron:copy && cd electron && npm run electron:start",
         "prettier:check": "prettier . --check",
-        "prettier:fix": "prettier . --write"
+        "prettier:fix": "prettier . --write",
+        "lib:update": "rm -rf node_modules/megashark-lib && rm package-lock.json && npm install",
```

Comment:
I would name them `megashark:*` instead. `lib` sounds very generic, you get the impression that you're updating a bit of everything.